### PR TITLE
Small fixes hp markup.tex

### PR DESCRIPTION
**Pull request by entorb**

### CODE REVIEW

File: chapters/hpmor-chapter-005.tex [hunk 1]
```diff
@@ -123,7 +123,7 @@ \chapter{The Fundamental Attribution Error}
 
 “And \emph{I} am pleased to learn that I shall be treated with the respect due to the family of Malfoy,” the other boy lobbed back, accompanied by a smile such as the highest of kings might bestow upon the least of his subjects, if that subject were honest, though poor.
 
-Eh…Damn, Harry was having trouble thinking up his next line. Well, everyone \emph{did} want to shake the hand of Harry Potter, so—“When my clothes are fitted, sir, might you deign to shake my hand? I should wish nothing more to put the capper upon this day, nay, this month, indeed, my whole lifetime.”
+Eh…Damn, Harry was having trouble thinking up his next line. Well, everyone \emph{did} want to shake the hand of Harry Potter, so— “When my clothes are fitted, sir, might you deign to shake my hand? I should wish nothing more to put the capper upon this day, nay, this month, indeed, my whole lifetime.”
```
Comment (rrthomas) [inline chapters/hpmor-chapter-005.tex:126]: Please remove this space!

File: chapters/hpmor-chapter-005.tex [hunk 2]
```diff
@@ -137,9 +137,9 @@ \chapter{The Fundamental Attribution Error}
 
 Harry coughed. “Actually, sorry, I’ve got no idea who you are really.”
 
-“\emph{Oh come on!}” the boy said with fierce disappointment. “Why’d you go and do that, then?” Draco’s eyes widened with sudden suspicion. “And how do you \emph{not} know about the Malfoys? And what are those \emph{clothes} you’re wearing? Are your parents \emph{Muggles?}”
+“\emph{Oh come on!}” the boy said with fierce disappointment. “Why’d you go and do that, then?” Draco’s eyes widened with sudden suspicion. “And how do you \emph{not} know about the Malfoys? And what are those \emph{clothes} you’re wearing? Are your parents \emph{Muggles}?”
 
-“Two of my parents are dead,” Harry said. His heart twinged. When he put it that way—“My other two parents are Muggles, and they’re the ones that raised me.”
```
Comment (rrthomas) [inline chapters/hpmor-chapter-005.tex:142]: Please remove this space!

**Comment by rrthomas** [comment]
Looks like a bunch of extra spaces need to be reverted in this PR!

**Comment by entorb** [comment]
Damn, I need to double-check all branches/perform a replace all on all branches

**Pull request closed without merging**